docker.d/passwords.yml

```diff
@@ -12,43 +12,13 @@ in:
       # dep-passwords.json
       'ENV == "dev" || (ENV == "fake-prod" && (COT_PRODUCT == "firefox" || COT_PRODUCT == "thunderbird"))':
         $let:
-
-          firefox_and_thunderbird_nonprod_nomac:
-            $map:
-              - 'signing7.srv.releng.mdc1.mozilla.com:9110'
-              - 'signing8.srv.releng.mdc1.mozilla.com:9110'
-              - 'signing9.srv.releng.mdc1.mozilla.com:9110'
-              - 'signing10.srv.releng.mdc2.mozilla.com:9110'
-              - 'signing11.srv.releng.mdc2.mozilla.com:9110'
-              - 'signing12.srv.releng.mdc2.mozilla.com:9110'
-            each(url):
-              - {"$eval": "url"}
-              - {"$eval": "SIGNING_SERVER_USERNAME"}
-              - {"$eval": "SIGNING_SERVER_PASSWORD"}
-              - ['gpg', 'sha2signcode', 'sha2signcodestub', 'sha2signcode-v2', 'sha2signcodestub-v2', 'mar', 'widevine', 'widevine_blessed']
-              - 'signing_server'
-
-          firefox_and_thunderbird_nonprod_mac:
-            $map:
-              - 'mac-v2-signing1.srv.releng.mdc1.mozilla.com:9110'
-              - 'mac-v2-signing3.srv.releng.mdc1.mozilla.com:9110'
-              - 'mac-v2-signing4.srv.releng.mdc2.mozilla.com:9110'
-              - 'mac-v2-signing5.srv.releng.mdc2.mozilla.com:9110'
-              - 'mac-v2-signing6.srv.releng.mdc2.mozilla.com:9110'
-              - 'mac-v2-signing8.srv.releng.mdc1.mozilla.com:9110'
-              - 'mac-v2-signing9.srv.releng.mdc1.mozilla.com:9110'
-              - 'mac-v2-signing10.srv.releng.mdc1.mozilla.com:9110'
-              - 'mac-v2-signing11.srv.releng.mdc2.mozilla.com:9110'
-              - 'mac-v2-signing12.srv.releng.mdc2.mozilla.com:9110'
-              - 'mac-v2-signing13.srv.releng.mdc2.mozilla.com:9110'
-            each(url):
-              - {"$eval": "url"}
-              - {"$eval": "SIGNING_SERVER_USERNAME"}
-              - {"$eval": "SIGNING_SERVER_PASSWORD"}
-              - ['macapp']
-              - 'signing_server'
-
           firefox_and_thunderbird_nonprod_autograph:
+            - ["https://autograph-external.prod.autograph.services.mozaws.net",
```

Comment:
Hm, so just to confirm, even though this is `dev/fake-prod` we're still talking to prod Autograph, just like we currently have in [puppet](https://github.com/mozilla-releng/build-puppet/blob/master/modules/signing_scriptworker/templates/dep-passwords.json.erb#L20). I see we only use the staging Autograph instance for `autograph_stage_mar384` format in puppet. Okay, makes sense 👍

Reply:
That's correct. The only time we should talk to stage autograph is to help the autograph team test out their changes.

Reply:
Sounds good